Makefile

```diff
@@ -0,0 +1,11 @@
+OPERATOR_SDK_VERSION ?= v0.18.2
+IMAGE_REGISTRY ?= quay.io
```

Comment:
Shouldn't be here something like `quay.io/oyahud` ?

Reply:
Added